go.mod

```diff
@@ -2,7 +2,7 @@
 
 module github.com/heroku/cnb-shim
 
-go 1.18
+go 1.23.0
```

Comment:
Big jump, but then we should be using something receiving security updates, so worth a try 🤷